src/DetailPage.js

```diff
@@ -15,6 +48,43 @@ export default function DetailPage() {
       <p>
         {game.description}
       </p>
+      <form onSubmit={handleSubmit}>
+        <h2>Update board game</h2>
+        <label>
+            Title
+          <input required name='title' onChange={e => setTitle(e.target.value)}/>
```

Comment:
example of my above comment to load in with the values to the input:
`<input required value={game.title} name='title' onChange={e => setGame({ ...game, title: e.target.value)}/>`